sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/ControlFlowGraph/ControlFlowGraphBuilder.cs

```diff
@@ -474,6 +477,20 @@ private void BuildLabeledStatement(LabeledStatementSyntax labeledStatement)
             currentBlock = CreateBlock(currentBlock);
         }
 
+        private void BuildTryStatement(TryStatementSyntax tryStatement)

```

Comment:
Unuseful for this PR but shall we reorder the BuildXXXStatement methods alphabetically?